.gitignore

```
@@ -1,4 +1,5 @@
 /rust/target
+/rust/tools/target
```

Should we ignore all rust targets?

```suggestion
/rust/**/target
```